tcpdialer.go

```diff
@@ -153,6 +153,8 @@ type TCPDialer struct {
 	// }
 	Resolver Resolver
 
+	DNSCacheDuration time.Duration
```

Comment:
Can you add some documentation here that also mentions that it will default to `DefaultDNSCacheDuration`.

Reply:
Hey @erikdubbelboer,

Comment is added,

Thanks!